go/pkg/gen/entities/execution_requested_v1.go

```diff
@@ -18,7 +18,7 @@ import (
 // ExecutionRequestedV1 
 type ExecutionRequestedV1 struct {
   ExecutionId string `json:"execution_id"`
-  GroupId string `json:"group_id,omitempty"`
```

Comment:
This is interesting. How is it possible that it was already `GroupId` :thinking:
Only the `omitempty` was removed :eyes: 
@CDimonaco what do you think?

Reply:
Well, the field in the schema was already `group_id`, the error was in the `required`. That's why it only changed the `omitempty`, as now it understood as required (so it cannot be omitted)

Reply:
AHA! thanks!

Reply:
However there is a different behavior with the elixir counterpart :thinking: 
anyway, maybe it's not worth bothering, at least immediately.